contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogFilters.vue

```diff
@@ -1,233 +1,89 @@
 <template>
 
-  <div>
+  <div class="catalog-filters-wrapper">
+    <!-- Mobile: Filter button that opens SidePanelModal -->
     <KButton
-      v-if="$vuetify.breakpoint.xsOnly"
-      class="drawer-btn"
-      :text="$tr('searchText')"
-      :primary="true"
-      appearance="flat-button"
-      @click.stop="drawer = true"
+      v-if="isMobile"
+      class="filter-button-mobile"
+      :text="$tr('filterText')"
+      appearance="raised-button"
+      icon="filter"
+      @click="openSidePanel"
     />
-    <CatalogFilterBar />
-    <VNavigationDrawer
-      v-model="drawer"
-      :permanent="$vuetify.breakpoint.smAndUp"
-      app
-      disable-route-watcher
-      :clipped="$vuetify.breakpoint.smAndUp"
-      :right="isRTL"
-    >
-      <VContainer class="filters pa-3">
-        <VToolbar
-          v-if="$vuetify.breakpoint.xsOnly"
-          color="transparent"
-          flat
-          dense
-        >
-          <VSpacer />
-          <VBtn
-            icon
-            flat
-            style="text-align: right"
-            @click="drawer = false"
-          >
-            <Icon icon="clear" />
-          </VBtn>
-        </VToolbar>
-
-        <!-- Keyword search -->
-        <VTextField
-          v-model="keywordInput"
-          color="primary"
-          :label="$tr('searchLabel')"
-          box
-          clearable
-          data-test="keywords"
-          autofocus
-          @input="setKeywords"
-        />
-
-        <!-- Language -->
-        <LanguageFilter
-          v-model="languages"
-          :menu-props="menuProps"
-        />
-
-        <!-- License (attach to self to keep in notranslate class) -->
-        <MultiSelect
-          v-if="!libraryMode"
-          v-model="licenses"
-          :items="licenseOptions"
-          :label="$tr('licenseLabel')"
-        />
 
-        <!-- Formats (attach to self to keep in notranslate class) -->
-        <MultiSelect
-          v-model="kinds"
-          :items="kindOptions"
-          :label="$tr('formatLabel')"
-        />
-
-        <!-- Starred -->
-        <Checkbox
-          v-if="loggedIn"
-          v-model="bookmark"
-          :label="$tr('starredLabel')"
-        />
-
-        <!-- Includes -->
-        <div class="subheading">
-          {{ $tr('includesLabel') }}
-        </div>
-
-        <div :style="{ display: 'flex', alignItems: 'center' }">
-          <Checkbox
-            v-model="coach"
-            aria-describedby="tooltip-coach"
-            :label="$tr('coachLabel')"
-          />
-          <HelpTooltip
-            :text="$tr('coachDescription')"
-            maxWidth="250px"
-            tooltipId="tooltip-coach"
-          />
-        </div>
+    <!-- Desktop/Tablet: Permanent side panel as page section -->
+    <aside
+      v-if="!isMobile"
+      class="filter-panel-desktop"
```

Comment:
In general, We don't usually make differentiations between "mobile vs desktop", and use instead wording like "windowIsSmall" because a Mobile device could render a "windowIsLarge" layout if we rotate it and see the content in landscape mode, and a desktop device could render a "windowIsSmall" layout if the window is shrinked or the user display is splitted displaying two or more apps.

So, it'd be great if we could call all these properties in terms of windowIsSmall/windowIsMedium/windowIsLarge naming that are the ones exposed by the `useKResponsiveWindow` composable.